### PR TITLE
StyleCop. Nothing else.

### PR DESCRIPTION
**Pull request by duvaneljulien**

### CODE REVIEW

File: src/Chromamboo/Program.cs [hunk 1]
```diff
@@ -20,7 +20,8 @@ private static void Main(string[] args)
             string password = null;
             if (args.Length <= 1)
             {
-                Console.Write("Chromamboo bambooApiBaseUrl bitbucketApiBaseUrl presentationProviderName [username] [password]");
+                Console.Write(
```
Comment (salfab) [inline src/Chromamboo/Program.cs:23]: meh.

File: src/Chromamboo/Providers/Notification/AtlassianCiSuiteBuildStatusNotificationProvider.cs [hunk 2]
```diff
@@ -73,21 +71,19 @@ public void Register(string planKey)
             this.presentationService.Update(buildsDetails);
         }
 
-
-        private BuildDetail GetBuildDetails(JToken jToken)
+        private BuildDetail GetBuildDetails(JToken jsonToken)
         {
-            var key = jToken["key"].Value<string>();
+            var key = jsonToken["key"].Value<string>();
```
Comment (salfab) [inline src/Chromamboo/Providers/Notification/AtlassianCiSuiteBuildStatusNotificationProvider.cs:76]: OK,  yet a JToken is actually  thing.  It's not Hungarian notation ;)

**Review by salfab — APPROVED** [review]
Thank you sir.

File: src/Chromamboo/Providers/Notification/AtlassianCiSuiteBuildStatusNotificationProvider.cs (hunk 2 above)
Comment (duvaneljulien) [reply src/Chromamboo/Providers/Notification/AtlassianCiSuiteBuildStatusNotificationProvider.cs:76]: You know I'm lazy. It was much faster to edit the variable's name than editing our stylecop rules.

**Pull request merged**